pkg/apis/kops/cluster.go

```diff
@@ -77,6 +77,9 @@ type ClusterSpec struct {
 	// NetworkID is an identifier of a network, if we want to reuse/share an existing network (e.g. an AWS VPC)
 	NetworkID string `json:"networkID,omitempty"`
 
+	// Topology defines the type of network topology to use on the cluster - default public
+	Topology *TopologySpec `json:"topology,omitempty"`
```

Comment:
I think - just as in mainline k8s, you have to copy-paste API changes into v1alpha1 now also.

It's fine to add fields (as long as the default is consistent with the existing behaviour), but removing them or changing their meaning requires a new API version and we need to create custom conversion code, so let's avoid that for now if we can :-)